int32   lidar_ver
uint32  udp_pkt_len
uint8[1440] udp_raw_data
